apps/vr-tests/src/stories/SpinButton.stories.tsx

```diff
@@ -70,6 +70,7 @@ storiesOf('SpinButton', module)
 
 // The stories for label placement are separate since they don't need to include hover/click states
 storiesOf('SpinButton', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
   // FIXME: SB6 duplicates same story ID decorators
```

Comment:
Can you remove this comment (and similar) now that it's no longer relevant?

Reply:
cleaned up all of the temp fixes here. don't need  `modifyDeprecatedDecoratorStyles` anymore